pytorch_lightning/utilities/cli.py

```diff
@@ -376,17 +420,23 @@ def instantiate_trainer(self, config: Dict[str, Any], callbacks: List[Callback])
             config["callbacks"].append(config_callback)
         return self.trainer_class(**config)
 
-    def add_configure_optimizers_method_to_model(self) -> None:
+    def add_configure_optimizers_method_to_model(self, subcommand: Optional[str]) -> None:
         """
         Adds to the model an automatically generated ``configure_optimizers`` method.
 
         If a single optimizer and optionally a scheduler argument groups are added to the parser as 'AUTOMATIC',
         then a `configure_optimizers` method is automatically implemented in the model class.
         """
+        if subcommand is None:
+            optimizers_and_lr_schedulers = self.parser.optimizers_and_lr_schedulers
+        else:
+            action_subcommands = [a for a in self.parser._actions if isinstance(a, _ActionSubCommands)][0]
+            subcommand_parser = action_subcommands._name_parser_map[subcommand]
```

Comment:
@mauvilsa Is there a better way to access a subcommand parser of the main parser?

This works. Just asking :)

Reply:
I think there is no better way.